pycode/memilio-epidata/memilio/epidata/getCaseData.py

```diff
@@ -363,58 +448,114 @@ def get_case_data(read_data=dd.defaultDict['read_data'],
+        different file names are used.
+    - Following data is generated and written to the mentioned filename
+        - All infected (current and past) for whole germany are stored in "cases_infected"
+        - All deaths whole germany are stored in "cases_deaths"
+        - Infected, deaths and recovered for whole germany are stored in "cases_all_germany"
+        - Infected split for states are stored in "cases_infected_state"
+        - Infected, deaths and recovered split for states are stored in "cases_all_state"
+        - Infected split for counties are stored in "cases_infected_county(_split_berlin)"
+        - Infected, deaths and recovered split for county are stored in "cases_all_county(_split_berlin)"
+        - Infected, deaths and recovered split for gender are stored in "cases_all_gender"
+        - Infected, deaths and recovered split for state and gender are stored in "cases_all_state_gender"
+        - Infected, deaths and recovered split for county and gender are stored in "cases_all_county_gender(_split_berlin)"
+        - Infected, deaths and recovered split for age are stored in "cases_all_age"
+        - Infected, deaths and recovered split for state and age are stored in "cases_all_state_age"
+        - Infected, deaths and recovered split for county and age are stored in "cases_all_county_age(_split_berlin)"
+
+    @param read_data True or False. Defines if data is read from file or downloaded. Default defined in defaultDict.
+    @param file_format File format which is used for writing the data. Default defined in defaultDict.
+    @param out_folder Folder where data is written to. Default defined in defaultDict.
+    @param start_date Date of first date in dataframe. Default 2020-01-01.
+    @param end_date Date of last date in dataframe. Default defined in defaultDict.
+    @param impute_dates True or False. Defines if values for dates without new information are imputed. Default defined in defaultDict.
+    @param moving_average Integers >=0. Applies an 'moving_average'-days moving average on all time series
+        to smooth out effects of irregular reporting. Default defined in defaultDict.
+    @param split_berlin True or False. Defines if Berlin's disctricts are kept separated or get merged. Default defined in defaultDict.
+    @param rep_date True or False. Defines if reporting date or reference date is taken into dataframe. Default defined in defaultDict.
+    @param files List of strings or 'All' or 'Plot'. Defnies which files should be provided (and plotted). Default 'All'.
+    @param to_dataset bool True or False. Whether to return the dataframe as an object instead of json file.
+        If True - returns objects with dataframes
+        If False - write dataframes into files
+        Default defined in defaultDict.
+
+    @return None
+    """
+    raw_df = fetch_case_data(
+        read_data=read_data,
+        out_folder=out_folder,
+        **kwargs,
```

Comment:
I think its better to call `conf = gd.Conf` first and pass `conf` into the three functions.

Reply:
Make sense!